src/split-button.secondary-button.test.interactions.ts

```diff
@@ -1,38 +1,29 @@
-import './menu.button.js';
-import './menu.options.js';
-import { aTimeout, expect, fixture, html } from '@open-wc/testing';
+import './option.js';
+import { expect, fixture, html } from '@open-wc/testing';
 import { click } from './library/mouse.js';
 import SplitButtonSecondaryButton from './split-button.secondary-button.js';
 
 it('sets `menuOpen` when its menu is opened', async () => {
   const host = await fixture<SplitButtonSecondaryButton>(html`
     <glide-core-split-button-secondary-button label="Label">
-      <glide-core-menu-button label="Label"></glide-core-menu-button>
+      <glide-core-option label="Label"></glide-core-option>
     </glide-core-split-button-secondary-button>
   `);
 
-  await click(host);
-
-  // Wait for the Mutation Observer to pick up the attribute change.
```

Comment:
Removed the mutation observer. It's a leftover from before Menu had a "toggle" event. Split Button Secondary Button had to observe Menu's `open` attribute.